internal/repos/check_connection.go

```diff
@@ -0,0 +1,42 @@
+package repos
+
+import (
+	"net"
+	"net/url"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+// checkConnection will parse the rawURL and make a best effort attempt to obtain a hostname. It
```

Comment:
nit: usually docs are written in present simple
```suggestion
// checkConnection parses the rawURL and makes a best effort attempt to obtain a hostname. It
```

Reply:
I prefer it in its current form.

Google's technical writing guide suggests to use active voice over passive voice: https://developers.google.com/tech-writing/one/active-voice.

Reply:
I mostly mean that docs with present simple are more frequent in our code base and it's better to keep to one or more popular thing for the sake of uniformity :)

Reply:
I disagree. 😁

Active voice should be preferred and we should attempt to move towards that instead of continuing to stick to using passive voice. 

Active voice is simpler and more effective for the reasons mentioned in the guide I linked above.

Reply:
I think the official Go docs overrule the Google tech writing guideline: https://go.dev/blog/godoc

Also: `checkConnection prases the rawURL` is still an active voice. The difference is the future vs. present tense.

Reply:
Yeah, I was referring to Indra's comment 

> Google's technical writing guide suggests to use active voice over passive voice:

Reply:
Makes sense. Fixed.